pkg/sdn/plugin/pod_linux.go

```diff
@@ -367,22 +383,15 @@ func (m *podManager) update(req *cniserver.PodRequest) (uint32, error) {
 	if err != nil {
 		return 0, err
 	}
-	runtimeService, err := m.getRuntimeService()
-	if err != nil {
-		return 0, err
-	}
-
 	filter := &kruntimeapi.PodSandboxFilter{
 		LabelSelector: map[string]string{ktypes.KubernetesPodUIDLabel: string(pod.UID)},
 	}
-	podSandboxList, err := runtimeService.ListPodSandbox(filter)
+	podSandboxID, err := m.getPodSandboxID(filter, req)
```

Comment:
As I mentioned in the other PR, I think the sandbox ID detection stuff for update() should go into node.go or update.go.  Then we just pass the actual sandbox ID into the PodRequest which gets set up in node.go::Update(), and we'll get it already here.